source/opt/iterator.h

```diff
@@ -158,8 +160,9 @@ inline bool UptrVectorIterator<VT, IC>::operator<(
 }
 
 template <typename VT, bool IC>
-inline UptrVectorIterator<VT, IC> UptrVectorIterator<VT, IC>::InsertBefore(
-    Uptr value) {
+template <bool C>
```

Comment:
That's fairly reasonable. Using `IsConstForMethod` now. `IC` now is consistently a short version for `IsConst`.